.gitignore

```diff
@@ -4,4 +4,5 @@
 /.vagrant
 .vscode/
 cmd/s2i/debug
+.idea/
```

Comment:
If you are using intellij products to do development, you should really put the .idea in your global gitignore file, as it's not a project specific file/folder that is being ignored.

https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore

Reply:
ok